services/file_get_info.proto

```diff
@@ -32,75 +42,109 @@ import "query_header.proto";
      */
     FileID fileID = 2;
 }
 
 /**
- * Response when the client sends the node FileGetInfoQuery
+ * A response to a query for the metadata of a file in the HFS.
  */
 message FileGetInfoResponse {
     /**
-     * Standard response from node to client, including the requested fields: cost, or state proof,
-     * or both, or neither
+     * The standard response information for queries.<br/>
+     * This includes the values requested in the `QueryHeader`
+     * (cost, state proof, both, or neither).
      */
     ResponseHeader header = 1;
 
-
     message FileInfo {
         /**
-         * The file ID of the file for which information is requested
+         * A file identifier.
+         * <p>
+         * This SHALL be the identifier of a file that exists in HFS.<br/>
+         * This value SHALL identify the file that was queried.
          */
         FileID fileID = 1;
 
         /**
-         * Number of bytes in contents
+         * A size, in bytes, for the file.
          */
         int64 size = 2;
 
         /**
-         * The current time at which this account is set to expire
+         * An expiration timestamp.
+         * <p>
+         * The file SHALL NOT expire before the network consensus time
+         * exceeds this value.<br/>
+         * The file SHALL expire after the network consensus time
+         * exceeds this value.<br/>
          */
         Timestamp expirationTime = 3;
 
         /**
-         * True if deleted but not yet expired
+         * A flag indicating this file is deleted.
+         * <p>
+         * A deleted file SHALL have a size `0` and empty content.
          */
         bool deleted = 4;
 
         /**
-         * One of these keys must sign in order to modify or delete the file
+         * A KeyList listing all keys that "own" the file.
+         * <p>
+         * All keys in this list MUST sign a transaction to append to the
+         * file content, or to modify file metadata.<br/>
+         * At least _one_ key in this list MUST sign a transaction to delete
+         * this file.<br/>
+         * If this is an empty `KeyList`, the file is immutable, cannot be
+         * modified or deleted, but MAY expire. A `fileUpdate` transaction MAY
+         * extend the expiration time for an immutable file.
          */
         KeyList keys = 5;
 
         /**
-         * The memo associated with the file
+         * A short description for this file.
+         * <p>
+         * This value, if set, SHALL NOT exceed 100 bytes when encoded as UTF-8.
          */
         string memo = 6;
 
         /**
-         * The ledger ID the response was returned from; please see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-198.md">HIP-198</a> for the network-specific IDs. 
+         * A ledger identifier for the responding network.
+         * <p>
+         * This value SHALL identify the distributed ledger that responded to
+         * this query.
          */
```

Comment:
```suggestion
     * <p>
     * This is originally defined in HIP-198 and depends on network
     * configuration. The current values, as of Q1 2024, are
     * <dl>
     *   <dt>Mainnet</dt><dd>0x00</dd>
     *   <dt>Testnet</dt><dd>0x01</dd>
     *   <dt>Previewnet</dt><dd>0x02</dd>
     *   <dt>Undefined</dt><dd>0x03</dd>
     *   <dt>Reserved</dt><dd>0x04</dd>
     * </dl>
         */
```

Reply:
I avoided including that detail for two reasons (and if I missed something and left it in somewhere; I need to correct that):
1. It's in HIP-198, and probably best managed there
2. It is likely to change with the new ledgerID implementation, and having to
   then update it here (as well as every other query) seems unnecessary.

I could easily be persuaded to reverse that approach, however.  Thoughts?

Reply:
Ok, I agree. But, I copied this text from other files and it should be removed there too for consistency:

- consensus_topic_info.proto
- contract_get_info.proto
- crypto_get_info.proto
- get_account_details.proto
- schedule_get_info.proto
- token_get_info.proto
- token_get_nft_info.proto

Reply:
Absolutely.
I was sure I had fixed all of those, but apparently I was mistaken.
They're fixed now.